src/Compilers/CSharp/Portable/Symbols/UpdatedContainingSymbolLocal.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+#nullable enable
+
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed class UpdatedContainingSymbolLocal : WrappedLocalSymbol
```

Comment:
>WrappedLocalSymbol [](start = 57, length = 18)

Is this the only implementation of `WrappedLocalSymbol`? If so, consider merging the two classes.